frontend/src/metabase/entities/databases/forms.js

```diff
@@ -31,6 +31,21 @@ const DATABASE_DETAIL_OVERRIDES = {
       </div>
     ),
   }),
+  "tunnel-private-key": (engine, details) => ({
+    title: t`SSH private key to connect to the tunnel`,
+    placeholder: t`Paste the contents of an ssh private key here`,
+    type: "text",
+  }),
+  "tunnel-private-key-passphrase": (engine, details) => ({
+    title: t`Passphrase for the SSH private key`,
+  }),
+  "tunnel-auth-option": (engine, details) => ({
+    title: t`SSH Authentication`,
+    options: [
+      { name: t`SSH Key`, value: "ssh-key" },
+      { name: t`Password`, value: "password" },
+    ],
+  }),
```

Comment:
@robdaemon I don't remember why these needed to be in overrides. What wasn't working if the server sends them down?

Reply:
Oh, I did it for the translations. On the server side, it's evaluated at startup time, not per user session, so I don't think it can be localized properly.

Reply:
Oh yeah that's right. If there's no way to localize those on the server, then I guess we can remove the strings from the server and just leave the field name there. That would avoid the duplication.